code-of-conduct.md

```diff
@@ -0,0 +1,37 @@
+# OpenContainer's Code of Conduct
+
+Behave as a community member, follow the code of conduct.
+
+
+## Code of Conduct
+
+The OpenContainers community is made up of a mixture of professionals and volunteers from all over the world.
+
+When we disagree, we try to understand why.
+Disagreements, both social and technical, happen all the time and OpenContainers is no exception.
+It is important that we resolve disagreements and differing views constructively.
+
+This code of conduct applies both within project spaces and in public spaces when an individual is representing the project or its community.
+Participants should be aware of these concerns.
+
+We are committed to making participation in this project a harassment-free experience for everyone, regardless of level of experience, gender, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, religion, or nationality.
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery
```

Comment:
Okay, looked up grammar. The recommend that full sentences end with periods, but this list shouldn't have any. Need to remove it from the last item in the list, though.